README.md

```diff
@@ -145,3 +145,28 @@ Pretty-printed JSON metadata listing
 Query offset(s) by timestamp(s)
 
     $ kafkacat -b mybroker -Q -t mytopic:3:2389238523  mytopic2:0:18921841
+
+# Running in docker
+
+There is no official image build yet but you may run `docker build -t kafkacat .` or use for example `solsson/kafkacat@sha256:3075a6ca2d8431910cf01af13acd2ef9542c8ed48547733eb5c4321364ef0b66`.
+
+Example:
+```bash
+docker run --rm kafkacat --help
+echo 1 > example.log
+docker run --name test-producer -d -v $(pwd)/example.log:/logs/example.log --entrypoint /bin/bash --net=host kafkacat \
+  -c 'tail -f /logs/example.log | kafkacat -b mybroker -t logs -P'
```

Comment:
I just thought this was a more interesting use case, but maybe only because I was working with log aggregation in Kubernetes.